openwisp_notifications/tasks.py

```diff
@@ -25,3 +27,15 @@ def delete_obsolete_notifications(instance_app_label, instance_model, instance_i
         | Q(target_object_id=instance_id)
     )
     Notification.objects.filter(where).delete()
+
+
+@shared_task
+def delete_old_notifications():
```

Comment:
being able to pass the number of days as argument was a good thing, so the number of days can be easily changed from the settings